admin-guide/en/appendix-1.-create-a-protected-ojs-3-sandbox-staged-with-git.md

```diff
@@ -22,16 +22,16 @@ The README here: [https://github.com/pkp/ojs](https://github.com/pkp/ojs) has in
 
    ```
    cd <httpd-docs-folder>
-   BRANCH='stable-3_2_1'
-   git clone -b $BRANCH --single-branch https://github.com/pkp/ojs.git ./
+   BRANCH='stable-3_3_1'
```

Comment:
Let's use `3_3_0` instead of `3_3_1` here and elsewhere, since `3_3_1` hasn't been released or tested yet.